raw_data/preprocessed/africa/senegal_french.txt

```diff
@@ -0,0 +1,259 @@
+
+(49) L'étude d'impact s'insère dans une procédure déjà existante d'autorisation, d'approbation ou d'octroi de concession; les principaux acteurs qui interviennent dans la procédure de l'étude d'impact sur l'environnement sont le promoteur et les autorités compétentes. L'étude d'impact est établie à la charge du promoteur et soumise par lui au Ministère chargé de l'environnement qui délivre un certificat d'autorisation après avis technique de la Direction de l'environnement et des établissements classés.
+
+(50) Les différentes catégories d'activités et les ouvrages dont la réalisation ou l'exploitation nécessite une étude d'impact sont définis par décret sur rapport du ministre chargé de l'environnement. Le décret établit et révise la liste des activités, travaux et documents de planification pour lesquels les autorités publiques ne pourront décider, approuver ou autoriser des travaux sans disposer d'une étude d'impact leur permettant d'en apprécier les conséquences sur l'environnement Tout projet nouveau inscrit sur cette liste et demandant une autorisation d'exploitation doit présenter obligatoirement un dossier d'étude d'impact sur l'environnement.
```

Comment:
Should there be a period between "l'environnement" and "Tout".

Reply:
Yes, there should be. I will update.